5e-SRD-Monsters.json

```diff
@@ -138,6 +138,18 @@
         }
       }
     ],
+    "multiattack_options": {
```

Comment:
Should these live under the Multiattack action itself?

Reply:
I could go either way on that. So whatever you think is best for the repository.

Reply:
Probably under the action.

Reply:
Probably under the action.